solutions/standard/variables.tf

```diff
@@ -44,6 +44,15 @@ variable "machine_type" {
   }
 }
 
+variable "image_name" {
+   description = "Enter a valid image name for Wazi VSI"
```

Comment:
See my comments in quickstart variation for the same variable.

Reply:
Added default value of image.